CHANGELOG.md

```diff
@@ -28,6 +28,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
```

Comment:
Fwiw, I don't think you need to maintain this anymore thanks to release notes. Release notes cover the user-facing changes, and Git log covers the developer changes.

You could keep this file and point it to docs.quantum.ibm.com/api/qiskit-ibm-transpiler/release-notes (once it goes live)

Reply:
OK, thank you @Eric-Arellano . I added that link in [CHANGELOG.md](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/69/commits/3b0a265fb123e06bdb14dbf6fb46b3c0f72ad38e) so we don't need to update that file anymore